src/shims/foreign_items.rs

```diff
@@ -102,6 +103,39 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         }
     }
 
+    /// Lookup the body of a non-external foreign item.
+    fn lookup_non_external_foreign_item(
+        &self,
+        abi: Abi,
+        link_name: Symbol,
+    ) -> InterpResult<'tcx, Option<&'mir mir::Body<'tcx>>> {
+        let this = self.eval_context_ref();
+        let tcx = this.tcx.tcx;
+        let mut instance = None;
+        for cnum in iter::once(LOCAL_CRATE).chain(tcx.crates().iter().copied()) {
+            for &(symbol, _) in tcx.exported_symbols(cnum) {
```

Comment:
The `_` here is [`SymbolExportLevel`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_middle/middle/exported_symbols/enum.SymbolExportLevel.html). I'm not sure do I need to care about that?

Reply:
Hm, no idea... maybe leave a FIXME.